.chromatic/main.js

```diff
@@ -6,7 +6,6 @@ module.exports = {
   },
   stories: [
     '../packages/**/chromatic/**/*.stories.@(js|jsx|ts|tsx)',
-    '../packages/@react-spectrum/s2/stories/*.stories.@(js|jsx|mjs|ts|tsx)',
```

Comment:
removed this since we don't need to run chromatic on our s2 docs storybook anymore